### DISCUSSION TITLE
New feature: Split screen!

### DISCUSSION THREAD

**Discussion by gastmaier** [BIPES/BIPES#76] [Announcements]
Now you can divide the screen in two by right clicking any tab!

The usage is very simple:
- Left click: the tab will fill the primary space (left);
- Right click: the tab will fill the secondary space (right).

This brings way more versatility to the development, like having the terminal output always accessible:

![Screenshot from 2021-11-25 23-53-16](https://user-images.githubusercontent.com/2892061/143519585-404dd262-3564-466a-b497-3031d88974c6.png)

Or checking the board pinout:

![Screenshot from 2021-11-25 23-49-45](https://user-images.githubusercontent.com/2892061/143519426-040c64c9-39ee-4581-97cb-740e541a1c40.png)

And even running plot scripts (more about that later...):

![Screenshot from 2021-11-25 23-50-19](https://user-images.githubusercontent.com/2892061/143519431-72ac4166-645a-4efe-a881-1836e79d00aa.png)

**Comment by zimba64** [comment]
hola 
poderiam faser um mini tutorial de como executar o programa direto do microcontrolador  .
eu ja tentei de todos os jeitos que consegui imaginar  .
todos os lugares disem so .
sauvar com main .py   . 
mas a onde  .
ja perguntei em todo o lugar  .
ninguem responde .
obrigado e aguardo
